vol/forms.py

```diff
@@ -157,14 +148,10 @@ def clean_data_aniversario(self):
         return val
 
     def clean_estado(self):
-        val = self.cleaned_data['estado'].strip().upper()
-        ufs = dict(UFS_SIGLA)
-        if val not in ufs.keys():
-            raise forms.ValidationError(u'Faltou selecionar o estado.')
-        return val
+        return self.cleaned_data['estado']
```

Comment:
Se for para só deixar o retorno do valor em cleaned_data, melhor remover o método, com a ressalva que se alguém quiser submeter o form programaticamente com um estado incorreto, acho que vai gravar no banco. Quer experimentar?